array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,22 @@
+/* exported equal */
+
+// look at each item of first array, starting from beginning until end
+//  look at each item of second array, starting from the beginning until end
+//    if index of first array at first array is strictly not equal to index of first array at second array
+//      give back false
+//      else if, index of second array at first array is strictly not equal to index of second array at second array
+//      give back false
+// give back true
+
+function equal(first, second) {
+  for (var i = 0; i < first.length; i++) {
+    for (var j = 0; j < second.length; j++) {
+      if (first[i] !== second[i]) {
+        return false;
+      } else if (first[j] !== second[j]) {
+        return false;
+      }
+    }
```

Comment:
You can remove this inner loop. I also recommend adding a guard (`if` statement) to immediately return `false` before starting the loop if the arrays aren't even the same length.